reconciler/reconciler.go

```diff
@@ -228,4 +229,5 @@ func (r *reconciler) reconcile() {
 			}
 		}
 	}
+	log.Debug("Finished reconcile")
```

Comment:
do you only want this log on a successful run? or would it be better if you did a deferred call?

Reply:
I'm not sure what you mean - it never returns earlier. But we could do a defer to guard against that happening in the future, so I can change it.